### PR TITLE
Page toc made sticky and scrollable

### PR DESCRIPTION
**Pull request by PaarthAgarwal**
Addresses #9. 
After applying changes
image

### CODE REVIEW

**Comment by PaarthAgarwal** [comment]
@tbrlpld this one is ready for review now.

**Comment by PaarthAgarwal** [comment]
After adding padding to the top:

image

File: package-lock.json [hunk 1]
```diff
@@ -1952,6 +1952,20 @@
 			"integrity": "sha1-FQStJSMVjKpA20onh8sBQRmU6k8=",
 			"dev": true
 		},
+		"node_modules/fsevents": {
```
Comment (tbrlpld) [inline package-lock.json:1955]: **Question** Is this in here on accident? I think we should try keep this out of the PR if we don't need this.

**Review by tbrlpld — CHANGES_REQUESTED** [review]
Thank for this @PaarthAgarwal 

Just one open question, then it's good to go!

File: package-lock.json (hunk 1 above)
Comment (PaarthAgarwal) [reply package-lock.json:1955]: It happened automatically after making changes I'm scss and rebuilding the project

Comment (PaarthAgarwal) [reply package-lock.json:1955]: It happened in #166 as well

Comment (PaarthAgarwal) [reply package-lock.json:1955]: Now when I look at it more closely then what removed in #166 got restored in this one. I suggest you take this branch delete package-lock.json file and rebuild to see if these changes should be there or not.

Comment (tbrlpld) [reply package-lock.json:1955]: I am not sure I understand what you want me to do here? Can you not remove these changes from the PR? Or should that have never been removed?

Comment (PaarthAgarwal) [reply package-lock.json:1955]: Let's drop them for the time being. Maybe they shouldn't be there.

**Comment by PaarthAgarwal** [comment]
Happy to help

**Comment by PaarthAgarwal** [comment]
Missing changelog entry for this.

**Comment by tbrlpld** [comment]
> Missing changelog entry for this.

True. Can you open a PR to add it?

**Comment by PaarthAgarwal** [comment]
I'll do

**Pull request merged**